packages/tasks/src/tasks/object-detection/about.md

```diff
@@ -21,9 +21,9 @@ Object Detection models are used to count instances of objects in a given image,
 You can infer with Object Detection models through the `object-detection` pipeline. When calling the pipeline you just need to specify a path or http link to an image.
 
 ```python
-model = pipeline("object-detection")
-
-model("path_to_cat_image")
+model = pipeline(task="object-detection", model="facebook/detr-resnet-50", device=0)
```

Comment:
I'm a fan of keeping gaps between = signs @pcuenca I saw you removed one, which one is better?

Reply:
I personally like spaces too, but:
- We should use the same criteria for consistency, in the previous version of the doc it was mixed.
- Our style tools in transformers, diffusers and other libraries remove spaces in function invocation, so all our source code is like that. For example: https://github.com/huggingface/transformers/blob/89439fea6458d1a430c6dbcadb983937416090fd/src/transformers/generation/utils.py#L1376. I think we should use the same style across docs too.

Reply:
@pcuenca I just thought since we prioritize readability in this doc I find this more readable. no strong opinion though if you say that we should use it like so